integration.py

```diff
@@ -267,7 +267,21 @@ def test():
 
 
 def autotest_all():
-    logger.info("This has not been implemented yet")
+    logger.info("Killing all currently running services")
+    kill_all_services()
+    logger.info("Running all Company 2 services")
+    run_company(2)
+    logger.info("Testing default Company 2 configuration")
+    # Test function will call Selenium tests when implemented
+    test()
+    for service in ["backend", "algs1", "algs2"]:
+        logger.info(f"Swapping out {service}")
+        swap_service(service)
```

Comment:
Sorry I realized this will not work since we will need to change the environment variables. Could you edit the swap_service function so that it will change the ports in the environment vairables for the affected modules?

Reply:
I can't find company1's repo to see their env settings, could you send a link to their github? And so would I be updating these env variables e.g. for algs1 and 2?

From backend dockerfile:
#Algs 1 & 2 APIs 
ALGS1_API=https://c2algs1.onrender.com/generate
ALGS2_API=https://algs2.onrender.com/predict